static/css/style.css

```diff
@@ -39,8 +39,31 @@ body {
 -------------------------------------------------- */
 /* Not required for template or sticky footer method. */
 
+.login {
+	padding: 10px 0px 5px 45px; 
+}
+.h1 {
```

Comment:
See my comments below about h1, h2 etc. Dots are used for classes that we define, built-in elements don't need one.